drivers/vulkan/vulkan_context.cpp

```diff
@@ -2271,7 +2271,7 @@ Error VulkanContext::prepare_buffers() {
 				print_verbose("Vulkan: Early suboptimal swapchain.");
 				break;
 			} else if (err != VK_SUCCESS) {
-				ERR_BREAK_MSG(err != VK_SUCCESS, "Vulkan: Did not create swapchain successfully.");
+				ERR_BREAK_MSG(err != VK_SUCCESS, "Vulkan: Did not create swapchain successfully." + String(string_VkResult(err)));
```

Comment:
An error enum that comes after a dot seems a bit weird. I'd prefix with "Error code: " or similar.